fv3core/utils/corners.py

```diff
@@ -212,17 +212,15 @@ def definition(q: FloatField):
         func = (
             fill_corners_2cells_mult_x if num_fill == 2 else fill_corners_3cells_mult_x
         )
-        stencil = gtstencil(
-            definition=definition,
-            externals={"func": func},
+        stencil = gtstencil(externals={"func": func})(
+            definition,
```

Comment:
was this something caught by mypy? it seems like it was valid, just a different convention?

Reply:
This was because I changed gtstencil to have a consistent static typing scheme, so it has to be used in the same way everywhere. This usage is the same as:

```
@gtstencil(externals={"func": func})
def defintiion():
    ...
```

which as you know is how we use as a decorator.

Fixing this made most files pass mypy.